sklearn/model_selection/_search.py

```diff
@@ -703,10 +705,16 @@ def evaluate_candidates(candidate_params):
                                                        X, y,
                                                        train=train, test=test,
                                                        parameters=parameters,
+                                                       split_progress=(
+                                                           split_i, n_splits),
```

Comment:
nit, `split_idx` and `param_idx` 

also maybe `cand_idx` beause candidate is used instead of param